x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -334,13 +334,13 @@ export interface AlertEvent {
     start: number;
     thread?: ThreadFields[];
     uptime: number;
-    Ext: {
+    Ext?: {
       /*
        * The array has a special format. The entity_ids towards the beginning of the array are closer ancestors and the
        * values towards the end of the array are more distant ancestors (grandparents). Therefore
        * ancestry_array[0] == process.parent.entity_id and ancestry_array[1] == process.parent.parent.entity_id
        */
-      ancestry: string[];
+      ancestry?: string[];
```

Comment:
my understanding is that all fields and subfields are optional (and untyped) in ecs. maybe runtime validation would be a better solution for this issue.